README.md

```diff
@@ -4,14 +4,62 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(:get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+RSpec.describe MyApp do
+  describe "::visit_example" do
+    # Create a manager with the `:get_example` fixture started
+    let!(:manager) { WebMock::Fixtures::Manager.run([:get_example]) }
+
+    before(:example) do
+      # Make call to www.example.org
+      MyApp.visit_example()
+    end
+
+    it "should have been requested" do
+      # Access the `WebMock::RequestStub` that was started
+      stub = manager[:get_example]
+
+      # Assert that the `WebMock::RequestStub` has been requested
+      expect(stub).to(have_been_requested)
+    end
+  end
+end
+```
+
+As well, by subclassing `WebMock::Fixtures::Manager` you can easily separate and organize your test fixtures. For example:
+
+```ruby
+# Manager to store fixtures associated with Google
+class GoogleManager < WebMock::Fixtures::Manager
+end
+GoogleManager.register_fixture(:get_google, :get, %r{www.google.com}, :body => "Google")
```

Comment:
It feels a little weird to state a class only to end it. Is there a syntax where we could also allow something like:

```
class GoogleManager
  fixtures = {
    :get_google => {
      # Make sure terminology lines up with WebMock (e.g. verb, method)
      :verb => :get,
      :pattern => %{www.google.com},
      :body => "Google",
    }
  }
end
```